slider/src/components/SliderPlugin.vue

```diff
@@ -30,13 +35,10 @@ export default {
     }
   },
   render() {
-    const minValue = this.options.minValue
-      ? Number.parseFloat(this.options.minValue)
-      : undefined
+    const minValue = this.minValue
     const maxValue = this.options.maxValue
       ? Number.parseFloat(this.options.maxValue)
       : undefined
```

Comment:
```suggestion
      : 1
```

On a side note; it could be convenient to have default values for `minValue` and `maxValue`